docs/fastdds/use_cases/large_data/large_data.rst

```diff
@@ -142,13 +142,46 @@ For receiving sockets, the command is:
 Flow Controllers
 ----------------
 
-*eProsima Fast DDS* provides a mechanism to limit the rate at which the data is sent by a
-DataWriter.
-These controllers can be configured at DataWriter or DomainParticipant
-level.
-On the DomainParticipant the throughput controller is configured on the |DomainParticipantQos::wire_protocol-api|
-member function, while the :ref:`dds_layer_publisher_dataWriterQos` uses the
-|DataWriterQos::throughput_controller-api| member function.
+*eProsima Fast DDS* provides a mechanism to limit the rate at which the data is sent by a DataWriter.
+These controllers should be registered on the creation of the DomainParticipant using |FlowControllersQos|, and
+then referenced on the creation of the DataWriter using |PublishModeQosPolicy|.
+
+A new thread is spawned the first time a flow controller is referenced by an asynchronous DataWriter.
+This thread will be responsible for arbitrating the network output of the samples being transmitted by all the
+DataWriters referencing the same flow controller.
+
+Flow controllers should be given a name so they can later on be referenced by the DataWriters.
+A default, unlimited, |FIFO_SCHED_POLICY-api| flow controller is always available with name
+|FASTDDS_FLOW_CONTROLLER_DEFAULT|.
```

Comment:
Both links |FIFO_SCHED_POLICY-api| and |FASTDDS_FLOW_CONTROLLER_DEFAULT| seem to be broken.